### PR TITLE
feat: add eslint, prettier

### PR DESCRIPTION
**Pull request by AfganAbbas**
Hello, I added eslint and prettier configuration to the app. Also, I give the freedom of configuration or adding rules to you, it is just implementing base vue-eslint rules. Additionally, you can run scripts for both prettier and eslint. 
Resolves: #24

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,9 +28,13 @@
     "@vitejs/plugin-vue": "^3.2.0",
     "@vue/test-utils": "^2.2.4",
     "autoprefixer": "^10.4.11",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-vue": "^9.20.1",
```
Comment (franciscobmacedo) [inline package.json:33]: can you move this dependecies to dev dependecies?

**Review by franciscobmacedo — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (AfganAbbas) [reply package.json:33]: Yess, you are right, it was avoided from my attention

Comment (franciscobmacedo) [reply package.json:33]: sorry, github "smart" decisions on hiding some content made it look like they were under dependecies. looks good, thank you!! 🙏

**Review by franciscobmacedo — APPROVED** [review]
⭐

**Comment by c0rdeiro** [comment]
I think it should have a  [prettier ignore](https://prettier.io/docs/en/install) for the node_modules and dist folders

**Pull request merged**